app/controllers/curator/digital_objects_controller.rb

```diff
@@ -33,7 +36,15 @@ def update
     def digital_object_params
       case params[:action]
       when 'create'
-        params.require(:digital_object).permit!
+        params.require(:digital_object).permit(:ark_id, :created_at, :updated_at,
+                                               admin_set: [:ark_id],
+                                               is_member_of_collection: [:ark_id],
+                                               contained_by: [:ark_id],
+                                               metastreams: {
+                                                 descriptive: {},
```

Comment:
mainly did it for convenience since there are alot of attributes in the model. I can whitelist them individually if you prefer but i think its fine considering the factory service handles it.

Reply:
Yep, cool with me.